src/flask_debugtoolbar/__init__.py

```diff
@@ -122,7 +131,11 @@ def _default_config(self, app):
 
     def dispatch_request(self):
         """Modified version of Flask.dispatch_request to call process_view."""
-        req = _request_ctx_stack.top.request
+        if version_builder.parse(flask.__version__) >= version_builder.parse("2.2.0"):
+            req = request_ctx.request
+        else:
+            req = _request_ctx_stack.top.request
```

Comment:
Oh I see, you can't just alias the package import (`from flask.globals import _request_ctx_stack as request_ctx`) because the object's API changed as well...

Reply:
Yep I discovered that after the comment too.